.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -23,7 +23,7 @@ on:
 
 jobs:
   lte-integ-test-containerized:
-    if: github.event.workflow_run.conclusion == 'success' && (github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch')
+    if: (github.event.workflow_run.conclusion == 'success' && github.repository_owner == 'magma') || github.event_name == 'workflow_dispatch'
```

Comment:
The dispatch eventhas now successful workflow conclusion. So workflows could not be started manually from the github gui.